src/modules/Exchange.ts

```diff
@@ -180,14 +191,29 @@ export class Exchange {
       starkKey,
       positionId,
       zeroExResponseObject,
+      signature = Buffer.from('', 'utf8'),
     }: {
       humanMinUsdcAmount: string,
       starkKey: string,
       positionId: BigNumberable,
       zeroExResponseObject: ZeroExSwapResponse,
+      signature?: Buffer,
```

Comment:
This signature is only used for registering a user on the contract right? If so, can we rename it to something like `registerUserSignature`?

Reply:
great call